CMakeLists.txt

```diff
@@ -162,6 +162,7 @@ option(USE_OPENCV "Use OpenCV" OFF)
 option(USE_OPENMP "Use OpenMP for parallel code" ON)
 option(USE_PROF "Use profiling" OFF)
 option(USE_QNNPACK "Use QNNPACK (quantized 8-bit operators)" ON)
+option(USE_PYTORCH_QNNPACK "Use QNNPACK (quantized 8-bit operators)" OFF)
```

Comment:
Adding a new user-facing option.

Reply:
Can we have a different description compared to `USE_QNNPACK`? It may get confusing for the user.

Reply:
When do we plan to set this new macro to ON? Do we know if the tests pass with this on?

Reply:
Good point regarding a better description.  I had to set USE_PYTORCH_QNNPACK to true in a subsequent commit to make a bunch of failing tests pass.  I left USE_QNNPACK to ON but can change it if that's what we want.